domains/tylermwise.json

```diff
@@ -0,0 +1,13 @@
+{
+    "description": "tylermwise.is-a.dev domain",
+    "owner": {
+      "username": "tylermwise",
+      "email": "contact@teamtbm.org"
+    },
+    "record": {
+       "NAMESERVER": "maceio.ns.porkbun.com",
+       "NAMESERVER": "curitiba.ns.porkbun.com",
+       "NAMESERVER": "salvador.ns.porkbun.com",
+       "NAMESERVER": "fortaleza.ns.porkbun.com"
```

Comment:
We do not support NS records, please use a URL redirect or another type of record.

Reply:
Sorry! Could I do `"CNAME": "tylermwise.github.io"`